easybuild/easyconfigs/p/pybind11/pybind11-2.6.0-GCCcore-10.2.0.eb

```diff
@@ -9,7 +13,12 @@ toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
 
 source_urls = ['https://github.com/pybind/pybind11/archive/']
 sources = ['v%(version)s.tar.gz']
-checksums = ['90b705137b69ee3b5fc655eaca66d0dc9862ea1759226f7ccd3098425ae69571']
+patches = ['1d3b04e805c8a7dd4d8b68e2a311c75a66cae1d5.patch']
```

Comment:
@sassy-crick Please rename the patch file to something like `pybind11-2.6.0_fix-test-extra-whitespace.patch`